cmd/release.go

```diff
@@ -20,6 +20,12 @@ var release = cli.Command{
 	Usage:     "Create and publish a new release from the given tag",
 	ArgsUsage: "<tag>",
 	Action:    releaseCmd,
+	Flags: []cli.Flag{
+		cli.BoolTFlag{
+			Name:  "homebrew",
+			Usage: "Create a new Homebrew version",
+		},
```

Comment:
should the flag be --no-homebrew instead, to match your description?

Reply:
This is a True Flag, so it's true by default (if not specified). To not create a new homebrew release, you'd have to do `--homebrew=false`.

Reply:
Or did you mean the PR description?

Reply:
Nope i meant the flag, because I did not read it closely enough to see the `T` :)